### PR TITLE
[Fabric-Admin] Add API to commission local bridge within its own fabric 

### PR DESCRIPTION
**Pull request by yufengwangca**
This PR is refactored from https://github.com/project-chip/connectedhomeip/pull/34990 to make review easier.

In E2E (End-to-End), each side consists of a Fabric-Admin and a Fabric-Bridge. The Fabric-Bridge needs to be commissioned to both sides to conduct fabric sync. Therefore, each Fabric-Admin needs to manage two bridges: one within its own fabric and one from the remote ecosystem. 

In commercial hub products, such as HomePod and Google Home, which host the fabric bridge, the hub is typically self-commissioned to its own fabric during startup

### CODE REVIEW

File: examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h [hunk 1]
```diff
@@ -31,7 +35,7 @@ class FabricSyncAddBridgeCommand : public CHIPCommand, public CommissioningDeleg
         AddArgument("device-remote-ip", &mRemoteAddr);
     }
 
-    void OnCommissioningComplete(NodeId deviceId, CHIP_ERROR err) override;
+    void OnCommissioningComplete(chip::NodeId deviceId, CHIP_ERROR err) override;
```
Comment (tehampson) [inline examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h:38]: nit, does this need to change?

File: examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.cpp [hunk 2]
```diff
@@ -161,6 +157,114 @@ CHIP_ERROR FabricSyncRemoveBridgeCommand::RunCommand()
...
+    DeviceMgr().PairLocalFabricBridge(deviceId);
+
+    return CHIP_NO_ERROR;
+}
+
+void FabricSyncRemoveLocalBridgeCommand::OnDeviceRemoved(NodeId deviceId, CHIP_ERROR err)
+{
+    if (mLocalBridgeNodeId != deviceId)
+    {
+        ChipLogProgress(NotSpecified, "An non-bridge device: NodeId: " ChipLogFormatX64 " is removed.", ChipLogValueX64(deviceId));
```
Comment (saurabhst) [inline examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.cpp:221]:
```suggestion
        ChipLogProgress(NotSpecified, "A non-bridge device: NodeId: " ChipLogFormatX64 " is removed.", ChipLogValueX64(deviceId));
```

**Review by tehampson — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

File: examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h (hunk 1 above)
Comment (yufengwangca) [reply examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h:38]: As convention, we don't 'using namespace ::chip;' in the .h, we need to add 'chip::'

File: examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.cpp [hunk 3]
```diff
@@ -161,6 +157,114 @@ CHIP_ERROR FabricSyncRemoveBridgeCommand::RunCommand()
...
+        return CHIP_NO_ERROR;
+    }
+
+    PairingCommand * pairingCommand = static_cast<PairingCommand *>(CommandMgr().GetCommandByName("pairing", "already-discovered"));
+
+    if (pairingCommand == nullptr)
+    {
+        ChipLogError(NotSpecified, "Pairing already-discovered command is not available");
+        return CHIP_ERROR_NOT_IMPLEMENTED;
+    }
```
[the remark this answers is not in the document]
Comment (yufengwangca) [reply examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.cpp]: Acked, we can make it more restrict

**Pull request merged**